packages/cardano-services/test/ChainHistory/__snapshots__/ChainHistoryHttpService.test.ts.snap

```diff
@@ -4,65 +4,39 @@ exports[`ChainHistoryHttpService healthy state /blocks/by-hashes server and snap
 Array [
   Object {
     "confirmations": 1957883,
-    "date": Object {
-      "__type": "Date",
-      "value": 1595995096000,
-    },
+    "date": 2020-07-29T03:58:16.000Z,
     "epoch": 74,
     "epochSlot": 27480,
-    "fees": Object {
-      "__type": "bigint",
-      "value": "0",
-    },
+    "fees": 0n,
     "header": Object {
       "blockNo": 1598507,
       "hash": "7a48b034645f51743550bbaf81f8a14771e58856e031eb63844738ca8ad72298",
       "slot": 1625880,
     },
-    "nextBlock": Object {
-      "__type": "undefined",
-    },
-    "previousBlock": Object {
-      "__type": "undefined",
-    },
+    "nextBlock": undefined,
+    "previousBlock": undefined,
```

Comment:
We might want to add some consecutive blocks to our database snapshot to test this

Reply:
ADP-2096